src/server/scripts/Spells/spell_generic.cpp

```diff
@@ -4884,6 +4884,39 @@ class spell_gen_shriveling_gaze : public AuraScript
     }
 };
 
+// 38048 - Curse of Pain
+enum CurseOfPain
+{
+    SPELL_CURSE_OF_PAIN           = 38048,
+};
+
+class spell_gen_curse_of_pain : public AuraScript
+{
+    PrepareAuraScript(spell_gen_curse_of_pain);
+
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_CURSE_OF_PAIN });
+    }
+
+    void OnPeriodic(AuraEffect const* /*aurEff*/)
+    {
+        Unit* target = GetTarget();
+        if (target && target->ToPlayer())
+        {
+            if (target->GetHealthPct() < 50.f)
+            {
+                target->RemoveAurasDueToSpell(SPELL_CURSE_OF_PAIN);
+            }
+        }
+   }
```

Comment:
this bracket is missplaced

Reply:
by like one space

Reply:
I have been staring at it for like a minute now cuz it looked off xD

Reply:
Well, it's in the name of the spell.